WidgetBus/Models/Network/BusClient.swift

```diff
@@ -155,6 +233,18 @@ class BusClient {
                 }
         }
 
+    class func getCityCodeList (
+        completion: @escaping ([CityCodeInfo], Error?) -> Void) {
+            _ = taskForGETRequest(
```

Comment:
https://github.com/DeveloperAcademy-POSTECH/WidgetBus/pull/17/files#r1028913757 
return 값을 사용하지 않는다면 삭제가 가능해보여요 ~
```suggestion
            taskForGETRequest(
```

Reply:
이거 lint 보기 싫어서 저렇게 만들었습니다. 반환값 있는데 안쓴다고 워닝이 떠요.

Reply:
@shwoghk14